.github/pull_request_template.md

```diff
@@ -1,11 +1,3 @@
 ### Description of Work carried through
 
 Brief description of what/why/how.
-
-#### Check list 
```

Comment:
Maybe a question of should we keep this in as a checklist to think about dependencies.  Appreciate if nobody uses it, but then question maybe we should?  There's some valid items in there that save time before building in Jenkins